tests/utils.py

```diff
@@ -78,17 +85,18 @@ def create_node_population(filepath, pop_name, circuit=None, node_sets=None, pop
         pop_name (str): population name inside the file.
         circuit (Mock/Circuit): either a real circuit or a Mock containing the nodes.
         node_sets: (Mock/NodeSets): either a real node_sets or a mocked node_sets.
+        pop_type (str): optional population type.
     Returns:
         NodePopulation: return a node population.
     """
     node_pop_config = {
         "nodes_file": filepath,
         "node_types_file": None,
-        "populations": {},
+        "populations": {pop_name: {}},
```

Comment:
is the `pop_name` key now mandatory?

Reply:
Population(s) are now required in the sonata extension documented at https://sonata-extension.readthedocs.io/en/latest/sonata_config.html :
> The populations dictionary should contain only the node populations that are part of the circuit. It must contain at least one population, but it doesn’t need to contain all the populations present in the nodes_file if there are more. It cannot be empty because it would mean that it’s incomplete (ie: a population that should have been added, wasn’t), or that the nodes_file shouldn’t be added to the circuit.